plugin-check.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * Plugin Name: Plugin Check
+ * Plugin URI: https://github.com/wordpress/plugin-check
```

Comment:
Because the plugin is already installed in the 10up repository, we should update it. When it moves to the WordPress repo, then we will update this. Make sense?
 
```suggestion
 * Plugin URI: https://github.com/10up/plugin-check
```

Reply:
@mukeshpanchal27 in the notes for the PR I stated that I opted for the WordPress namespace instead of 10up as we will move this across later. Whilst the repository is hosted on the 10up organisation, this repo won't be made available to use until it moved across to the WordPress organisation. Because of this, I felt it was best to keep it as WordPress to save us changing this later down the line.

I think this is fine to keep as is.

Reply:
Make sense. Thanks.